sar_pipeline/pipelines/isce3_rtc/metadata/stac.py

```diff
@@ -213,6 +215,23 @@ def _extract_http_link(self, text: str) -> str:
         url_match = re.search(r"https?://\S+", text)
         return url_match.group() if url_match else None
 
+    def _get_dem_type_from_dem_description(self, dem_description):
+        dem_type_matches = [
+            dem_type for dem_type, url in DEM_TYPE_URL.items() if url in dem_description
+        ]
+
+        if len(dem_type_matches) == 1:
+            return dem_type_matches[0]
+        else:
+            logging.error(
+                "Could not determine a unique DEM type from the URL in the "
+                f"DEM description. Description: {dem_description}. "
+                f"DEM type URL map: {DEM_TYPE_URL}. Matches: {dem_type_matches}"
+            )
+            raise ValueError(
+                f"Expected exactly one DEM type match, found {len(dem_type_matches)}."
```

Comment:
Is it worth saying what the matches were? Or is it sufficient to be told there was an incorrect number of matches?